config/all-in-one/values.yaml

```diff
@@ -50,11 +50,8 @@ rookceph:
 # -----------------------------------------------------------------------------
 cp4waiops:
 
-  # Specify whether or not to install CP4WAIOps.
-  enabled: true
```

Comment:
Can we add a `enabled` param in aiManager section if remove this one ?

Reply:
Will be addressed in a separate PR